HW01 - Jenkins container/Official Jenkins/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM openjdk:8-jdk
+
+RUN apt-get update && apt-get install -y git curl && rm -rf /var/lib/apt/lists/*
+
+ARG user=jenkins
+ARG group=jenkins
+ARG uid=1000
+ARG gid=1000
+ARG http_port=8080
+ARG agent_port=50000
+
+ENV JENKINS_HOME /var/jenkins_home
+ENV JENKINS_SLAVE_AGENT_PORT ${agent_port}
```

Comment:
Lines 5-13: Define variables for building image. Define user and usergroup with GID and UID = 1000. As far as I remember, UID and GID = 1000 gets first created user. Interesting, what will be if another user created earlier? Maybe, it will be better to use another numbers, like 900, which is used for docker group in Linux.